packages/components/src/ListView/Items/Items.component.js

```diff
@@ -37,6 +37,11 @@ export class ItemsComponent extends React.PureComponent {
 		this.renderToggleAll = this.renderToggleAll.bind(this);
 
 		this.hasToggleAll = this.props.showToggleAll && this.props.items.length > 1;
+		this.cache = new CellMeasurerCache({
+			defaultHeight: props.getItemHeight(),
+			fixedWidth: true,
+			keyMapper: () => 1,
```

Comment:
With that all rows will have the same height everytime. You trigger a measure(), that will calculate and cache the height with the key `1`. All other rows will then be mapped with `1` too, and have the same size.
Is that what you want ?